metadata/water/stream_gauges_nhd.md

```diff
@@ -0,0 +1,76 @@
+# Title
+
+Utah Stream Gauges NHD
+
+## ID
+
+a3e384d1-eaaa-4887-a9f7-f49ab60ea6f4
+
+## Brief Summary
+
+Points dataset of USGS gauges used to measure watercourse output.
+
+## Summary
+
+This dataset is derived from the [National Hydrography Dataset](https://www.usgs.gov/national-hydrography/national-hydrography-dataset) (NHD) and has been modified by UGRC to provide additional functionality for Utah-based users.
+
+<!--- Other than clipping to the state boundary, did we modify this dataset in some way like we have for other NHD datasets? --->
```

Comment:
Stream gauges were selected from the original NHDPoint dataset. Unfortunately we decreased the functionality of the source data by only providing a subset of it.